Gordon360/Services/PosterCleanupService.cs

```diff
@@ -0,0 +1,40 @@
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Gordon360.Services;
+using System.Linq;
+
+
+
+public class PosterCleanupService : BackgroundService
+{
+    private readonly IServiceProvider _services;
+
+    public PosterCleanupService(IServiceProvider services)
+    {
+        _services = services;
+    }
```

Comment:
I believe you can use a primary constructor here
```suggestion
public class PosterCleanupService(IServiceProvider services) : BackgroundService
{
```